README.md

```diff
@@ -8,7 +8,7 @@ Tuil is a template that helps to get you started with React Native using TypeScr
 **Note this template supports react-native ^0.60.0 and react-native-cli ^2.0.0, because of breaking changes in the custom template mechanism.**
 
 ```bash
-npx react-native init MyApp --template=react-native-template-tuil
+npx react-native init MyApp --template=tuil
```

Comment:
This no longer works with the latest version of the react-native cli if i am correct?

Reply:
Not sure. It couldn't resolve `--template=react-native-template-tuil`, but could resolve `--template=tuil`.

Reply:
Are you using the old CLI? It was an issue with the old CLI. Since the new CLI it is not possible anymore to pass a shorthand.

Reply:
Ah ok, I'm using:

```
$ react-native -v
react-native-cli: 2.0.1
```